src/features/venue/api/__mocks__/useVenueOffers.ts

```diff
@@ -1,5 +1,27 @@
 import { VenueOffersResponseSnap } from 'features/venue/fixtures/venueOffersResponseSnap'
+import { venueResponseSnap } from 'features/venue/fixtures/venueResponseSnap'
+import { LocationType } from 'features/search/enums'
 
 export const useVenueOffers = jest
   .fn()
   .mockReturnValue({ data: { hits: VenueOffersResponseSnap, nbHits: 3 } })
+
+  const venueId = venueResponseSnap.id
```

Comment:
```suggestion
const venueId = venueResponseSnap.id
```

+ newline à la fin du fichier.

tu peux linter ce fichier ?